addons/editor-compact/userstyle.css

```diff
@@ -0,0 +1,492 @@
+  width: 2rem;
+  height: 2rem;
+  bottom: 0.25rem;
+}
+[class*="sprite-selector-item_sprite-image-inner"] {
+  margin: 0;
+}
+img[class*="sprite-selector-item_sprite-image"] {
+  max-width: 56px;
+  max-height: 32px;
+}
+
+[class*="action-menu_main-button"] {
+  width: 2rem;
+  height: 2rem;
+}
+[class*="action-menu_main-button"] img {
+  width: 1.5rem;
+  height: 1.5rem;
+}
+[class*="action-menu_more-buttons-outer"] {
+  width: 2rem;
+  bottom: 2rem;
+  padding-bottom: calc(2rem / 2);
+  margin: 0 0 calc(2rem / -2) 0;
+  border-top-left-radius: 2rem;
+  border-top-right-radius: 2rem;
+}
+button[class*="action-menu_more-button"] {
+  width: 2rem;
+  height: 2rem;
+}
+
+/* Tabs area */
+[class*="gui_tab-list"] {
+  height: 2rem;
+}
+[class*="gui_tab_"] {
+  padding-left: 1rem;
+  padding-right: 1rem;
+  border-radius: 0.5rem 0.5rem 0 0;
+}
+[class*="gui_tab_"] > img {
+  display: none;
+}
+
+#s3devToolBar {
+  align-self: center;
+}
+.s3devLabel {
+  align-items: center;
+}
+#s3devFind {
+  height: 20px;
+}
+#s3devHelpContent {
+  height: calc(100% - 2rem);
+}
+
+/* Code tab */
+[iscolumns="true"] .blocklyToolboxDiv,
+.blocklyFlyout {
+  width: 210px;
+}
+[class*="gui_tab-panel"]:not([iscolumns="true"]) .sa-flyout-placeHolder {
+  width: calc(210px + 1px);
+}
+.sa-lock-image {
+  left: calc(210px - 30px);
+}
+.blocklyScrollbarVertical.blocklyFlyoutScrollbar {
+  transform: translate(calc(196.5px + 60px), 2.5px) !important;
+}
+[iscolumns="true"] .blocklyScrollbarVertical.blocklyFlyoutScrollbar {
+  transform: translate(196.5px, 172.5px) !important;
+}
+.scratchCategoryItemBubble,
+.scratchCategoryItemIcon {
+  width: 0.75rem;
+  height: 0.75rem;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 0.6rem 0.6rem;
+}
+.scratchCategoryItemIcon {
+  background-size: 0.75rem 0.75rem;
+}
+[class*="gui_tab-panel"]:not([iscolumns="true"]) .blocklyToolboxDiv {
+  height: calc(100% - 2rem) !important;
+  scrollbar-width: none;
+}
+[class*="gui_tab-panel"]:not([iscolumns="true"]) .blocklyToolboxDiv::-webkit-scrollbar {
+  display: none;
+}
+[class*="gui_tab-panel"]:not([iscolumns="true"]) [class*="gui_extension-button-container_"] {
+  height: 2rem;
+}
+[iscolumns="true"] [class*="gui_extension-button-container_"] {
+  width: calc(210px - 2px - 1rem);
+}
+
+/* Assets panel */
+[class*="selector_wrapper"] {
+  width: 100px;
+}
+[class*="selector_list-item"] {
+  margin: 0.25rem auto;
+}
+[class*="selector_list-area"] {
+  overflow-y: auto;
+}
+[class*="selector_list-area"] > div:first-child {
+  margin-top: 0.25rem;
+}
+[class*="selector_list-area"] [class*="sprite-selector-item_delete-button"] {
+  top: -0.5rem;
+}
+[class*="selector_list-area"] img[class*="sprite-selector-item_sprite-image"] {
+  max-width: 52px;
+  max-height: 44px;
+}
+#react-tabs-5 [class*="selector_list-area"] > div > div {
+  height: 2rem;
+}
+#react-tabs-5 [class*="sprite-selector-item_sprite-image-outer"] {
+  display: none;
```

Comment:
Scratch 2.0 did have these icons so I don't think this addon should remove them.

Reply:
It does make the sounds list more compact, but perhaps it would be better to be consistent with the costumes tab.